casper/src/main/scala/coop/rchain/casper/api/BlockAPI.scala

```diff
@@ -810,7 +812,9 @@ object BlockAPI {
   }
 
   def exploratoryDeploy[F[_]: Monad: EngineCell: Log: SafetyOracle: BlockStore](
-      term: String
+      term: String,
+      blockHash: Option[String] = none,
+      usePreStateHash: Boolean = false
```

Comment:
Please add documentation comment for this method with description for parameters. Also this can be part of PR description which is necessary for release notes.

TODO: We should investigate what is easiest way to generate some API specification like Open API.

Reply:
create an issue for that https://github.com/rchain/rchain/issues/2974